apps/aesophia_http/src/aesophia_http_handler.erl

```diff
@@ -264,24 +259,11 @@ handle_request('ApiVersion', _Req, #{ spec := Spec }) ->
 handle_request('Api', _Req, #{ spec := Spec }) ->
     {200, [], jsx:decode(Spec, [return_maps])}.
 
-to_old_aci_item_def(C0) ->
-    #{ typedefs := Typedefs } = C0,
-    C1 = maps:remove(typedefs, C0),
-    C1#{ type_defs => Typedefs }.
-
-to_old_aci_item(#{contract := Contract}) ->
-    #{contract => to_old_aci_item_def(Contract)};
-
-to_old_aci_item(#{namespace := Namespace}) ->
-    #{namespace => to_old_aci_item_def(Namespace)}.
-
 generate_aci(Contract, Options) ->
     Opts = compile_options(Options),
-    try aeso_aci:contract_interface(json, Contract, Opts) of
+    try aeso_aci:contract_interface(json, Contract, Opts ++ [{no_code, true}]) of
```

Comment:
In `aesophia_cli` we explicitly require a `--no-code` option to enable this, but I guess this is fine (you could possibly get an ACI for non-compileable code, but that wouldn't be a big problem I think).